tests/fixtures/api/batches/_.POST.json

```diff
@@ -13,6 +13,8 @@
     "user_id": 1,
     "webhook": "10.0.1.5",
     "configuration": {"dt": 0.1, "precision": "normal"},
+    "start_datetime": "2021-11-10T15:27:06.698066",
```

Comment:
We can probably add this to the "jobs" fixture as well!

Reply:
Done!